run-benchmark.sh

```diff
@@ -959,6 +977,7 @@ checkBenchResults() {
         fi
 	    echo "$date|$bench|$iter|$status|$errorCode||$elapsed" >> ${LOGBASE}logs/test_run.csv
     done
+    cp ${LOGBASE}logs/test_run.csv ${LOGDIR}
```

Comment:
test_run.csv is created for Jenkins and it's linked with other similar files.
Don't repurpose it.